index.html

```diff
@@ -10,951 +12,1033 @@
             Would you like to get the latest news?
-          </div>
-          <div class="box">
-            <div class="field is-grouped">
-              <p class="control is-expanded">
-                <input class="input">
-              </p>
-              <p class="control">
-                <a class="button is-info">Notify Me</a>
-              </p>
-            </div>
-          </div>
+					</h2>
+          <form aria-labelledby="subscribtion" class="box">
+            <fieldset class="field is-grouped">
+							<legend class="visuallyhidden">You can enter email address for email subscribtion and click "Notify Me" button.</legend>
+              <label for="emailInput" class="control is-expanded visuallyhidden">Enter your email</label>
+              <input aria-labelledby="subscribtion" name="emailInput" id="emailInput">
+              <button aria-labelledby="subscribtion" id="subscribeButton" name="subscribeButton" type="submit" class="button is-info">Notify Me</button>
+            </fieldset>
+          </form>
         </div>
       </div>
-    </div>
-  </div>
+    </section>
+  </header>
+	<main id="maincontent">
+		<div id="crumbs_frame" class="container" aria-label="breadcrumbs">
+			<ul class="bct">
+				<!-- .bct / Breadcrumbs -->
+				<li><a href="/">W3C</a><span class="cr">»</span></li>
+				<li><a href="/Consortium/">Test page</a><span class="cr">»</span></li>
+				<li><a href="" class="current" aria-current="page">About Us</a></li>
+			</ul>
+		</div>
+		<article class="section" id="about">
+			<section class="section-heading has-text-centered">
+				<h2 class="title is-2">About Us</h2>
+				<h3 class="subtitle is-5">We of all trades, master of "some"</h3>
+				<div class="container">
+					<p>Web developers with more than <b>4 years</b> of well-rounded experience with a degree in the
+						field of
+						<b>Computer Science</b>, extensive knowledge of modern Web techniques and love for <b>Coffee</b>.
+						Looking for an opportunity to work and upgrade, as well as being involved in an organization that
+						believes
+						in gaining a competitive edge and giving back to the community.</p>
+				</div>
+			</section>
 
-  <div class="section" id="about">
-    <div class="section-heading has-text-centered">
-      <div class="title is-2">About Us</div>
-      <div class="subtitle is-5">We of all trades, master of "some"</div>
-      <div class="container">
-        <p>Web developers with more than <b>4 years</b> of well-rounded experience with a degree in the
-          field of
-          <b>Computer Science</b>, extensive knowledge of modern Web techniques and love for <b>Coffee</b>.
-          Looking for an opportunity to work and upgrade, as well as being involved in an organization that
-          believes
-          in gaining a competitive edge and giving back to the community.</p>
-      </div>
-    </div>
+			<section id="maincontent" class="content container">
```

Comment:
As I see the tag `<main` (line 94) has the same `id="maincontent"`

Reply:
it's true, already fixed, thank you